jmix-translations/content/io/jmix/businesscalendarflowui/messages.properties

```diff
@@ -0,0 +1,89 @@
+#
+# Copyright 2021 Haulmont.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Menu config
+menu-config.buscal_BusinessCalendarModel.list=Business calendars
```

Comment:
fix message key according to review on https://github.com/jmix-framework/jmix-premium/pull/153